pkg/systemd/hw-detect.js

```diff
@@ -23,6 +23,8 @@ import * as machine_info from "machine-info.js";
 const InfoDMIKey = {
     version: "product_version",
     name: "product_name",
+    alt_version: "board_vendor",
```

Comment:
This looks wrong.  THis mixes "version" and "vendor", is that intended?

Reply:
There's also a board_version? But that one seems more likely to be undefined